src/main/java/games/strategy/triplea/ai/weakAI/WeakAI.java

```diff
@@ -809,8 +809,8 @@ public void purchase(final boolean purchaseForBid, final int PUsToSpend, final I
       defUnitsAtAmpibRoute = amphibRoute.getEnd().getUnits().getUnitCount();
     }
     final Resource PUs = data.getResourceList().getResource(Constants.PUS);
-    final int totPU = player.getResources().getQuantity(PUs);
-    int leftToSpend = totPU;
+    final int totPu = player.getResources().getQuantity(PUs);
```

Comment:
`totPu` seems a bit over abbreviated. 
PU is part of the nomenclature, so IMO it is an appropriate abbreviate. That suggests to me: `totalPu` as a reasonable alternative to `totPu`